ciao-cli/instance.go

```diff
@@ -377,26 +378,75 @@ func (cmd *instanceListCommand) run(args []string) error {
 				fmt.Fprintf(w, "\tN/A")
 				fmt.Fprintf(w, "\tN/A\n")
 			}
-
 			w.Flush()
 		} else {
 			fmt.Printf("Instance #%d\n", i+1)
-			fmt.Printf("\tUUID: %s\n", server.ID)
-			fmt.Printf("\tStatus: %s\n", server.Status)
-			fmt.Printf("\tPrivate IP: %s\n", server.Addresses.Private[0].Addr)
-			fmt.Printf("\tMAC Address: %s\n", server.Addresses.Private[0].OSEXTIPSMACMacAddr)
-			fmt.Printf("\tCN UUID: %s\n", server.HostID)
-			fmt.Printf("\tImage UUID: %s\n", server.Image.ID)
-			fmt.Printf("\tTenant UUID: %s\n", server.TenantID)
-			if server.SSHIP != "" {
-				fmt.Printf("\tSSH IP: %s\n", server.SSHIP)
-				fmt.Printf("\tSSH Port: %d\n", server.SSHPort)
-			}
+			dumpInstance(&server)
 		}
 	}
 	return nil
 }
 
+type instanceShowCommand struct {
+	Flag     flag.FlagSet
+	instance string
+}
+
+func (cmd *instanceShowCommand) usage(...string) {
+	fmt.Fprintf(os.Stderr, `usage: ciao-cli [options] instance show [flags]
```

Comment:
well, this is the generic form.
below the flags are specified. but i can change it if you want.

The show flags are:

  -instance string
    	Instance UUID